packages/model-viewer/src/three-components/ARRenderer.ts

```diff
@@ -122,6 +125,22 @@ export class ARRenderer extends EventDispatcher {
     super();
     this.threeRenderer = renderer.threeRenderer;
     this.threeRenderer.xr.enabled = true;
+
+    this.xrLight = new XREstimatedLight(this.threeRenderer);
+
+    this.xrLight.addEventListener('estimationstart', () => {
+      if (!this.isPresenting) {
+        return;
+      }
+
+      const scene = this.presentedScene!;
+      scene.add(this.xrLight!);
```

Comment:
nit: can we put the xrLight null check up with the return statement so we can get rid of all these bangs?

Reply:
Thanks. I'm not familiar with TypeScript yet. Your comment is very helpful. Updated.